src/services/backend/workers/background/service.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
better use this with `next line` not global

Reply:
my counter-offer is to disable this rule completely, it's more headache then benefit